### PR TITLE
fix: maximize file import card when git import card is not available

### LINKED ISSUE #11558: [Task]: Git sync UI polishing
### SubTasks

 - [ ] This can use some UI/UX improvements. The problem is with the formatting of the error messages, specially the dual alert icon. image

 - [x] The hover label takes 2-3 seconds to show. This should be instant. Screenshot 2022-02-28 at 1 33 44 PM

 - [ ] The repo disconnection today feels like we can add a new repo link after deleting. Whereas we completely delete the git data from that app. This needs some UX improvements. Screenshot 2022-02-28 at 1 17 58 PM BE0C48BD-BB28-4204-8D2B-31F462E6A4B5

 - [x] When you open branch window, everything in the background should be covered with a translucent layer. Shifting the focus on the branches, its a crucial action to switch branch. Also we need to remove BETA tag from here. Screenshot 2022-02-28 at 1 39 49 PM

 - [ ] This error pops up when someone clicks remote branches. Screenshot 2022-02-28 at 1 40 32 PM

 - [ ] The hover states for modal are not there. Please follow the new modal design. 
5CD63F22-611C-411A-A0AA-98E417447596

 - [x] The edit navigation (link) needs to be removed. We can simply say go to your profile and edit default. Also when you toggle default configuration the Author name and Author email goes blank. We can make these editable and existing information can still be there.  B53C82EA-FA93-4376-97C1-CEAF96074DB3

 - [x] The placeholder for new repo url can be some dummy format hinting about the SSH repo url. B414615D-0E23-4D39-8BAE-1FE8D6CB7049

 - [ ] The git errors should point towards the field which can be fixed to correct this. This can use some design help.  08FFDA18-D778-40C8-B279-79308ADA8AED

 - [x] Placeholder for commit input is missing
 - [x] Bug on resize of the commit input.
 - [x] The time for latest deployed preview link should say “less than 1 min ago” instead of 0 seconds ago.
 - [x] When commit number is 9+ things break.

### PR DESCRIPTION
**Pull request by haojin111**
## Description

> Updated tooltip of "commit" in the bottom bar as "Commit" from "Commit Changes"
> Without gitimport feature, import from file picker is fulfilled over import application modal.

Fixes #11558 , #12029 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

> Manual test

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:white_circle: Total coverage has not changed</summary>
No changes to code coverage between the base branch and the head branch</details>

### CODE REVIEW

File: app/client/src/pages/Applications/ImportApplicationModal.tsx [hunk 1]
```diff
@@ -74,8 +74,8 @@ const Row = styled.div`
   justify-content: space-between;
 `;
 
-const FileImportCard = styled.div`
-  width: 320px;
+const FileImportCard = styled.div<{ gitEnabled?: boolean }>`
+  width: ${(props) => (props.gitEnabled ? "320px" : "100%")};
```
Comment (IAmAnubhavSaini) [inline app/client/src/pages/Applications/ImportApplicationModal.tsx:78]: This is not we agreed upon. Hide the GitImportCard when git is not enabled.

**Review by IAmAnubhavSaini — CHANGES_REQUESTED** [review]

File: app/client/src/pages/Applications/ImportApplicationModal.tsx (hunk 1 above)
Comment (haojin111) [reply app/client/src/pages/Applications/ImportApplicationModal.tsx:78]: yes, when we hide git import card, file import card should be full width.

**Review by IAmAnubhavSaini — APPROVED** [review]

File: app/client/src/ce/constants/messages.test.ts [hunk 2]
```diff
@@ -72,7 +72,7 @@ describe("messages", () => {
 
 describe("git-sync messages", () => {
   const expectedMessages = [
-    { key: "COMMIT_CHANGES", value: "Commit changes" },
+    { key: "COMMIT_CHANGES", value: "Commit" },
```
Comment (IAmAnubhavSaini) [inline app/client/src/ce/constants/messages.test.ts:75]: Not required. Design has changed again to "Commit N changes" which I am working on. You have to revert this change.

File: app/client/src/ce/constants/messages.ts [hunk 3]
```diff
@@ -620,7 +620,7 @@ export const DEPLOY_TO_CLOUD = () => "Deploy to cloud";
 export const DEPLOY_WITHOUT_GIT = () =>
   "Deploy your application without version control";
 export const DEPLOY_YOUR_APPLICATION = () => "Deploy your application";
-export const COMMIT_CHANGES = () => "Commit changes";
+export const COMMIT_CHANGES = () => "Commit";
```
Comment (IAmAnubhavSaini) [inline app/client/src/ce/constants/messages.ts:623]: Not required. Design has changed again to "Commit N changes" which I am working on. You have to revert this change.

**Review by IAmAnubhavSaini — CHANGES_REQUESTED** [review]

**Review by IAmAnubhavSaini — APPROVED** [review]

**Comment by RakshaKShetty** [comment]
Import modal looks as below. Its fine for now 
image

**Review by IAmAnubhavSaini — DISMISSED** [review]

**Comment by IAmAnubhavSaini** [comment]
@haojin111  failing cypress test

```

────────────────────────────────────────────────────────────────────────────────────────────────────
                                                                                                    
  Running:  Smoke_TestSuite/ClientSideTests/GitSync/Git_spec.js                           (9 of 269)
  Estimated: 4 minutes, 32 seconds


  Git sync:
    ✓ shows remote is ahead warning and conflict error during commit and push (100944ms)
    ✓ detect conflicts when merging head to base branch (43309ms)
    ✓ supports merging head to base branch (16994ms)
    ✓ enables pulling remote changes from bottom bar (49465ms)
    1) "after all" hook for "clicking '+' icon on bottom bar should open deploy popup"


  4 passing (4m)
  1 failing

  1) Git sync:
       "after all" hook for "clicking '+' icon on bottom bar should open deploy popup":
     CypressError: Timed out retrying after 20000ms: `cy.type()` failed because this element:

`<div class="bp3-editable-text bp3-disabled t--application-name editable-application-name max-w-48">...</div>`

is being covered by another element:

`<div class="bp3-overlay-backdrop bp3-overlay-appear-done bp3-overlay-enter-done" tabindex="0"></div>`

Fix this problem, or use {force: true} to disable error checking.

https://on.cypress.io/element-cannot-be-interacted-with

Because this error occurred during a `after all` hook we are skipping the remaining tests in the current suite: `Git sync:`

Although you have test retries enabled, we do not retry tests when `before all` or `after all` hooks fail
      at $Cy.ensureDescendents (https://dev.appsmith.com/__cypress/runner/cypress_runner.js:160723:87)
      at ensureDescendents (https://dev.appsmith.com/__cypress/runner/cypress_runner.js:147550:8)
      at ensureDescendentsAndScroll (https://dev.appsmith.com/__cypress/runner/cypress_runner.js:147557:14)
      at ensureElIsNotCovered (https://dev.appsmith.com/__cypress/runner/cypress_runner.js:147691:5)
      at runAllChecks (https://dev.appsmith.com/__cypress/runner/cypress_runner.js:147878:52)
      at retryActionability (https://dev.appsmith.com/__cypress/runner/cypress_runner.js:147901:16)
      at tryCatcher (https://dev.appsmith.com/__cypress/runner/cypress_runner.js:13212:23)
      at Function.Promise.attempt.Promise.try (https://dev.appsmith.com/__cypress/runner/cypress_runner.js:10486:29)
      at tryFn (https://dev.appsmith.com/__cypress/runner/cypress_runner.js:165383:61)
      at whenStable (https://dev.appsmith.com/__cypress/runner/cypress_runner.js:165422:14)
      at https://dev.appsmith.com/__cypress/runner/cypress_runner.js:164909:18
      at tryCatcher (https://dev.appsmith.com/__cypress/runner/cypress_runner.js:13212:23)
      at Promise._settlePromiseFromHandler (https://dev.appsmith.com/__cypress/runner/cypress_runner.js:11147:31)
      at Promise._settlePromise (https://dev.appsmith.com/__cypress/runner/cypress_runner.js:11204:18)
      at Promise._settlePromise0 (https://dev.appsmith.com/__cypress/runner/cypress_runner.js:11249:10)
      at Promise._settlePromises (https://dev.appsmith.com/__cypress/runner/cypress_runner.js:11329:18)
  From Your Spec Code:
      at Context.eval (https://dev.appsmith.com/__cypress/tests?p=cypress/integration/Smoke_TestSuite/ClientSideTests/GitSync/Git_spec.js:289:47)



[mochawesome] Report HTML saved to /home/runner/work/appsmith/appsmith/app/client/results/mochawesome_001.html


  (Results)

  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ Tests:        5                                                                                │
  │ Passing:      4                                                                                │
  │ Failing:      1                                                                                │
  │ Pending:      0                                                                                │
  │ Skipped:      0                                                                                │
  │ Screenshots:  1                                                                                │
  │ Video:        true                                                                             │
  │ Duration:     3 minutes, 55 seconds                                                            │
  │ Estimated:    4 minutes, 32 seconds                                                            │
  │ Spec Ran:     Smoke_TestSuite/ClientSideTests/GitSync/Git_spec.js                              │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘

```

Screenshot 2022-03-23 at 6 05 46 PM

**Review by IAmAnubhavSaini — CHANGES_REQUESTED** [review]
Screenshot 2022-03-25 at 1 39 07 PM

**Review by IAmAnubhavSaini — APPROVED** [review]
Fixed merge issues.

**Pull request merged**